slm_porosity/problem.inp

```diff
@@ -141,6 +141,8 @@ powder_depth = 1.0
 convective_transfer = 2.434e-05
 radiative_transfer = 3.510e-04
 absolute_temperature = 2.148e-01
+scanning_strategy = F
+trajectory_file = 'trajectory.txt'          # file structure: x_i y_i z_i vect(x_i) vect(y_i) vect(z_i) (coordinate) (direction)
```

Comment:
What does your comment mean?  Use `# line format: x1 x2 [y1 y2] [z1 z2]`

Reply:
So we use x1 y1 z1 x2 y2 z2 format?

Reply:
1. I suggest another format. See my comment carefully.
2. I do not agree that our format should be close to STL. I think it should be convenient for us in the current moment and the nearest perspective for you master thesis, not more. My suggestion is based on the analysis of your current `trajectory.txt`.